curriculum/challenges/english/14-responsive-web-design-22/learn-responsive-web-design-by-building-a-piano/612ebcba99bfa46a15370b11.md

```diff
@@ -7,7 +7,19 @@ dashedName: step-27
 
 # --description--
 
-Now you need to make it responsive. Add a `@media` query with a `max-width` of `768px`.
+The `@media` at-rule, also known as a media query, is used to conditionally apply CSS. Media queries are commonly used to apply CSS based on the viewport width using the `max-width` and `min-width` properties.
+
+In the below example the padding is applied to the `.card` class when the viewport is `960px` wide and below.
+
+```css
+@media (max-width: 960px) {
```

Comment:
Two questions 

1) Do we want to demo min-width and ask for max-width?

2) Do you think users will randomly insert the CSS rule for the card class? Do we need to add a test for that?

Reply:
1. I think it is better not to confuse people and keep the example using `max-width`
2. It doesn't really matter what selector they put inside. It works with or without a selector inside the media query. The test doesn't check for that and it isn't part of the requirement. I'm fine with them being able to just copy-pasting the example and change the px value in the condition to pass the step.